java/algorithm/src/test/java/org/apache/arrow/algorithm/sort/TestFixedWidthInPlaceVectorSorter.java

```diff
@@ -209,4 +212,29 @@ public void testChoosePivotAllPermutes() {
       }
     }
   }
+
+  @Test
+  public void testSortInt2() {
+    try (IntVector vector = new IntVector("vector", allocator)) {
+      ValueVectorDataPopulator.setVector(vector,
+          0, 1, 2, 3, 4, 5, 30, 31, 32, 33,
+          34, 35, 60, 61, 62, 63, 64, 65, 6, 7,
+          8, 9, 10, 11, 36, 37, 38, 39, 40, 41,
+          66, 67, 68, 69, 70, 71);
+
+      FixedWidthInPlaceVectorSorter sorter = new FixedWidthInPlaceVectorSorter();
+      VectorValueComparator<IntVector> comparator = DefaultVectorComparators.createDefaultComparator(vector);
+
+      sorter.sortInPlace(vector, comparator);
+
+      String actual = "[" + String.join(
+          ", ", IntStream.range(0, vector.getValueCount()).mapToObj(
+              i -> String.valueOf(vector.get(i))).collect(Collectors.toList())) + "]";
+
+      assertEquals(
+          "[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, " +
```

Comment:
It's a little strange to construct a String for this assertion. I think that it would be better to collect the values into a `int[]` array and use `Assertions#assertArrayEquals`.

Reply:
Revised the code accordingly. Thanks for the good suggestion.